Bottom: d30755e036498d42963af7a8bcf23198710b4938
Top:    fd0aebb2e8eebf572a2da40480e4c524fcec2357
Author: scott snyder <sss@karma>
Date:   2019-12-27 00:17:48 -0500

fix for cyanide



---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java
index 2a8ed6f..4916950 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java
@@ -82,7 +82,7 @@ public class Cyanide extends RandomIndexedComic {
 			throw e;
 		}
 		final_str = final_str.replaceAll(".*comics%2F","");
-		final_str = final_str.replaceAll("%2F.*","");
+		final_str = final_str.replaceAll("\".*","");
 		Log.d(Cyanide, "In parseForLatestId, final_str = "+final_str);
 		return Integer.parseInt(final_str);
 	}
